Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM islandoracollabgroup/isle-ubuntu-basebox:1.1.1
+FROM adoptopenjdk/openjdk8:latest
 
 ENV INITRD=no \
     ISLANDORA_UID=${ISLANDORA_UID:-1000} \
     ENABLE_XDEBUG=${ENABLE_XDEBUG:-false} \
     PULL_ISLE_BUILD_TOOLS=${PULL_ISLE_BUILD_TOOLS:-true} \
-    ISLE_BUILD_TOOLS_REPO=${ISLE_BUILD_TOOLS_REPO:-https://github.com/Islandora-Collaboration-Group/isle_drupal_build_tools.git} \
+    ISLE_BUILD_TOOLS_REPO=${ISLE_BUILD_TOOLS_REPO:-https://github.com/Born-Digital-US/ISLE-Drupal-Build-Tools.git} \
```

Comment:
Hi Noah, 
We need to change this back to ICG repo. 
Thanks!
Bethany

Reply:
commit added below 0025cb1